scripts/mainsite.js

```diff
@@ -0,0 +1,213 @@
+var lastStreamer, existingIframe;
+
+window.addEventListener("message", (event) => {
+  if (event.data.eventName == "UPDATE_STATE" && event.data.params.quality && !document.hidden)
+    if (/^((?:160|360|480|720|900|960|1080)p(?:30|48|50|60)|chunked)$/.test(event.data.params.quality))
+      localStorage.setItem("embAdbQuality", event.data.params.quality);
+});
+
+const untouchedPages = [(link) => link.endsWith("/directory"), (link) => link.includes("/clip/")];
+
+var observer = new MutationObserver(function (mutations, observer) {
+  var container = document.querySelector(".video-player .tw-absolute");
+
+  //Optimization: We do not check everything when numbers like viewers or anything updates.
+  if (mutations.length === 1 && (mutations[0].target.classList.contains("tw-animated-number--monospaced") || mutations[0].target.classList.contains("tw-animated-number")))

```

Comment:
nit: long line of code, I personally follow airbnb style guide for this.

```suggestion
  if (
    mutations.length === 1
    && (mutations[0].target.classList.contains("tw-animated-number--monospaced")
      || mutations[0].target.classList.contains("tw-animated-number")
    )
 )
```
or something like this